data/hai/hai reveal 4 middle.txt

```diff
@@ -1141,29 +1141,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Teeneep is allowed to respond and says ominously, "You know what will happen next."`
+			`	An angry Elder replies, "You wouldn't dare!" Teeneep yells back, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
+			`	Sayari leads you to a private meeting room.`
+			`	Teeneep explains her ominous words. "Our democracy allows a Petition of No Confidence against the Elders. If there are enough petitioners, it forces an election. Activists in the organization I founded have led a grassroots movement pushing a petition for weeks.`
+			`	"It was obvious what the Elder Council's response would be," and with a bit of sarcasm she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, my supporters arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`
+			`	Sayari assures you, "These petitions take time and rarely succeed. It is more of a warning to the Elders than anything else. Still, we should watch the situation.`
+			`	"For now," she says, "we're all in need of some lunch."`
+			choice
+				`	"I've heard good things about acorn pizza."`
+					goto pizza
+				`	"Do you two like spicy food?"`
+					goto spicy
+				`	"A burger would be great right about now."`
+			`	Teeneep leads you to a heavy metal pub; the music is audible from down the street. They have the best burgers you've ever tasted, even though the meat is synthetic, made from nuts.`
+				goto frantic
+			label spicy
+			`	Sayari takes you to her favorite restaurant, a famous Korean-Hai fusion place. She tells the waiter, "I won't leave happy unless my mouth is on fire." You try a bite from Sayari's plate, but can't swallow without gagging. She seems to enjoy the inferno of capsaicin.`
+				goto frantic
+			label pizza
+			`	You join Teeneep and Sayari at a fusion restaurant about a block down the road from the "Elders' Chambers" - which turns out to be the official name of the whole government complex. The pizza is as good as the rumors you've heard, even though the "cheese" is processed nut butter.`
+			label frantic
+			`	Not half way through her food, Sayari's data pad beeps frantically. She looks at it and mutters, "Oh dear, oh no." She lapses into Hai for a moment before returning. "The petition! It hit the trending issues board and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action, and it's only been an hour... this is what you hoped would happen, isn't it?"`
+			`	Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
+			`	Sayari's eyes are alive with thinly-veiled panic as she slams her hand down on the table. Teeneep seems shocked, as if she had never seen Sayari so emotional.`
```

Comment:
```suggestion
			`	Sayari's eyes are alive with thinly-veiled panic as she slams her paw down on the table. Teeneep seems shocked, as if she had never seen Sayari so emotional.`
```
I don't like using paw instead of hand, but it's more consistent with everything else.